src/webnn_native/openvino/ienn/src/ie_model.cpp

```diff
@@ -438,6 +438,21 @@ ie_operand_t* Model::AddRelu(ie_operand_t* input) {
   return CreateOperand(node_name);
 }
 
+ie_operand_t* Model::AddReduceMean(ie_operand_t* input,
+                                   ie_reduce_mean_options_t* options) {
+  auto input_node = name_node_map_[input->name];
+  SizeVector axes = ToVector(options->axes, options->axesCount);
+  auto axes_node = op::Constant::create(element::i64, Shape{axes.size()}, axes);
+  // Issue: When input rank == 3 and keeDimensions == false, reduceMean produces
+  // a result with wrong shape. This will be fixed by
+  // https://github.com/openvinotoolkit/openvino/pull/5986 in OpenVINO 2021/4.
+  auto reduce_mean_node = std::make_shared<op::v1::ReduceMean>(
```

Comment:
https://docs.openvinotoolkit.org/latest/openvino_docs_ops_reduction_ReduceMean_1.html
> Scalar or 1D tensor of type T_IND with axis indices for the 1st input along which reduction is performed. Accepted range is [-r, r-1] where where r is the rank of input tensor.

-1 is not the last dimension?

> all values must be unique, repeats are not allowed. Required.

Maybe we need to verify it, and double check it for DML.

Reply:
-1 is the last dimension in IE. And DML dosen't support -1 as last dimension, I have converted it when it's -1 at this [line](https://github.com/webmachinelearning/webnn-native/pull/13/files#diff-8c735e422d57ddc0234da2ed876a76e5bf398a447c94c4e819b14f9d8fd7c812R889). @fujunwei